gemrb/core/ScriptedAnimation.cpp

```diff
@@ -684,15 +684,15 @@ bool ScriptedAnimation::Draw(const Region &screen, const Point &Pos, const Color
 		flag |= BLIT_SEPIA;
 	}
 
-	if (Transparency & BLIT_TINTED) {
-		flag |= BLIT_TINTED;
-	}
-
 	if ((Transparency & IE_VVC_TINT)==IE_VVC_TINT) {
 		tint = p_tint;
 	}
+
 	ieDword flags = flag;
-	if (game) game->ApplyGlobalTint(tint, flags);
+	if (Transparency & BLIT_TINTED) {
+		flags = flag |= BLIT_TINTED;
+		if (game) game->ApplyGlobalTint(tint, flags);
+	}
```

Comment:
Again, mostly an educated guess, but otherwise - as for missiles - local effects are too dark at night.